content/200-concepts/100-components/02-prisma-client/240-metrics.mdx

```diff
@@ -20,23 +20,25 @@ Prisma Client provides the following metrics:
 
 - Counters (always increase):
 
-  - `query_total_queries`: The total number of database queries executed.
-  - `query_total_operations`: The total number of Prisma operations executed.
-    - A Prisma operation is a statement like `await prisma.user({ include: { posts: true }})`, which Prisma Client translates into one or more SQL queries.
-    - The value returned by `query_total_operations` can be less than `query_total_queries`, because some operations create multiple queries.
+  - `prisma_datasource_queries_total`: The total number of datasource queries executed.
+  - `prisma_client_queries_total`: The total number of Prisma Client queries executed.
```

Comment:
Wouldn't it make more sense to list the `client queries` first?
You have 1 client query in code that might do multiple datasource queries (sql queries or mongodb commands).

Reply:
This order makes more sense to me: introducing datasource queries first, then introducing client queries as potentially a set of datasource queries and explaining this in the sub-bullet

Reply:
As a user I write Prisma Client queries, I do not write SQL (datasource) queries. 
In tracing Prisma Client queries wrap around Datasource queries.
Focussing on datasource queries is database/SQL thinking (and wording). But we usually focus on the Prisma view.

Does that make sense as explanation for my concern here?

(Not super strong opinion here btw - but this just makes more sense to me)

Reply:
Got it, makes sense now. I've reordered them and tweaked the sub-bullet, agree that that does work better.